files/build_templates/telemetry.service.j2

```diff
@@ -10,7 +10,7 @@ StartLimitBurst=3
 
 [Service]
 User={{ sonicadmin_user }}
-ExecStartPre=/usr/local/bin/{{docker_container_name}}.sh start
-ExecStart=/usr/local/bin/{{docker_container_name}}.sh wait
-ExecStop=/usr/local/bin/{{docker_container_name}}.sh stop
```

Comment:
>docker_container_name [](http://example.com/codeflow?start=26&length=21)

Is it possible to keep the template in this j2 file, and render it for either telemetry container or gnmi container? You may need to use template var for "Description".